acceptance/tests/base/dsl/helpers/web_helpers/web_helpers_test.rb

```diff
@@ -0,0 +1,66 @@
+require 'helpers/test_helper'
+require 'webrick'
+require 'webrick/https'
+
+test_name 'dsl::helpers::web_helpers #link_exists?' do
+
+  http_cmd = "ruby -rwebrick -e'WEBrick::HTTPServer.new(:Port => 80, :DocumentRoot => Dir.pwd).start' > /tmp/mylogfile 2>&1 &"
+  host = hosts[0]
+  on(host, http_cmd)
+  on(host, 'mkdir tmp')
+  create_remote_file(host, 'tmp/test.txt', 'test')
```

Comment:
I Initially didn't use them as didn't have the ability to pass the -p flag to the mktemp command to specify a different tmp directory (it defaults to /tmp and the tmpdir method is hardcoded to only pass the -dt flag).
I was serving the current directory with webrick which was /root.
Anyway, I changed it to serve the /tmp directory and switched to use those methods instead.